packages/story-editor/src/components/keyboardShortcutsMenu/keyboardShortcutList.js

```diff
@@ -354,6 +354,16 @@ const shortcuts = {
             </kbd>
           ),
         },
+        {
+          label: __('Access floating menu', 'web-stories'),
```

Comment:
Still am not 100% with calling this a `Floating menu` as I think `Floating toolbar` fits a little better. But that is just me.

Reply:
like it!